wremnants/combine_theory_helper.py

```diff
@@ -283,6 +343,8 @@ def set_np_model(self, model):
         self.np_hist_name = self.corr_hist_name.replace("Corr", "FlavDepNP")
         self.np_hist = self.card_tool.getHistsForProcAndSyst(signal_samples[0], self.np_hist_name)
 
+        self.scale_hist_name = self.corr_hist_name.replace("Corr", "PtDepScales")
```

Comment:
You always want to use pt dependent scales? I think it should be an option

Reply:
If I understand the code correctly I think this will break for corrections that don't have the resume scales in them

Reply:
Currently both the pt-dependent and inclusive scales are added.

This requires the FO scales rather than the resummation ones, but yah I'll add a protection or option here.

Reply:
So this histogram is actually only used downstream in case setupCombine is run with --resumUnc scale which is not the default.

This relies on having the full set of scetlib scales which anyways aren't available in the older files.

There are other backwards incompatibilities related to the transition and FO scale variations with the default option however.  Are there specifically other correction files we want to support as the nominal at this stage?  (--theoryCorr scetlib_dyturboMSHT20an3lo seems to break also for other reasons related to ew uncertainties somehow?)

Reply:
Older files can now be used with the appropriate combination of ```--resumUnc tnp_minnlo``` or ```--resumUnc minnlo``` together with ```--noTransitionUnc```